### PR TITLE
List View related directives

### PR DESCRIPTION
**Pull request by vakrilov**

### CODE REVIEW

File: ng-sample/app/list-test.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                </ListView>   
+                         
+            </GridLayout>
+            
+            <Label row="2" text="-==END==-" fontSize="20"></Label>
+        </GridLayout>
+    `
+    // TEMPLATE WITH COMPONENT
+    // <template #item="item" #i="index" #odd="odd" #even="even">
+    //     <item-component [data]="item" [odd]='odd' [even]='even'></item-component>
```
Comment (NathanWalker) [inline ng-sample/app/list-test.ts]: Does using the `item-component` not work?

**Comment by vakrilov** [comment]
@NathanWalker 
It works - I have left it commented in the demo just for reference.

**Comment by NathanWalker** [comment]
I'd love to see this get merged. @vakrilov the changes likely just need to be rebased with latest master since the renderer changes with beta.1 and the renderer.

**Comment by vakrilov** [comment]
Yup - rebase in progress ...

**Comment by hdeshev** [comment]
Merged manually to master.

**Pull request closed without merging**